spring-cloud-skipper-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -418,7 +418,7 @@ include::{snippets}/list-documentation/list-releases-by-release-name/http-respon
 include::{snippets}/list-documentation/list-releases-by-release-name/response-fields.adoc[]
 
 [[resources-repositories]]
```

Comment:
This anchor link name can also be fixed though it doesn't matter. will fix while merging.